homeassistant-config/automations.yaml

```diff
@@ -1,33 +1,80 @@
-# Flip flop one light according to the status of another
-- id: '1674420359624'
-  alias: Flip Flop
+# Damper Automations
```

Comment:
@LucasKnezevich This diff looks super weird, all I did was remove one of my automations. I think because the structure between things is the same is why all the red shows up in different places.

Reply:
But by all means PLEASE check to make sure this looks right.